Makefile

```
@@ -278,7 +278,7 @@ git.dep: ${GOPATH}/src/istio.io/istio ${GOPATH}/src/istio.io/tools
 ${GOPATH}/bin/kind:
 	echo ${GOPATH}
 	mkdir -p ${TMPDIR}
-	go get -u sigs.k8s.io/kind
+	git clone https://github.com/kubernetes-sigs/kind ${GOPATH}/src/sigs.k8s.io/kind && cd ${GOPATH}/src/sigs.k8s.io/kind && git checkout v0.2.1 && go install .
```

another option if docker is available and HEAD is still desired is: 
```suggestion
	git clone https://github.com/kubernetes-sigs/kind && cd kind && make install
```
I do think we should instead get another kind release out and would request sticking to released versions for CI given the alpha state.

---

Thanks, let me have a try.